properties/basic.py

```diff
@@ -623,6 +669,26 @@ def error(self, instance, value, error=None, extra=''):
             )
         )
 
+    def deserialize(self, value):
+        """De-serialize the property value from JSON
+
+        If no deserializer has been registered, this converts the value
+        to the wrapper class with given dtype.
+        """
+        if self.deserializer:
+            return self.deserializer(value)
+        if value is None:
+            return None
+        return self.wrapper(value).astype(self.dtype[0])
+
+    @staticmethod
+    def to_json(value):
+        return value.tolist()
+
+    @staticmethod
+    def from_json(value):
```

Comment:
Ah, I see what you are saying. `Array` class still inherits `from_json` from `Property` so I think it makes sense to override it with more correct behaviour. Even though `deserialize` doesn't use it doesn't mean it won't be used elsewhere (for example, some other custom serializer that relies on all property's `from_json` methods?).